Tools/WebKitTestRunner/InjectedBundle/ios/AccessibilityControllerIOS.mm

```diff
@@ -99,10 +100,9 @@ static id findAccessibleObjectById(id obj, NSString *idAttribute)
 {
 }
 
-RefPtr<AccessibilityUIElement> AccessibilityController::accessibleElementById(JSStringRef idAttribute)
+RefPtr<AccessibilityUIElement> AccessibilityController::accessibleElementById(JSContextRef context, JSStringRef idAttribute)
 {
-    auto page = InjectedBundle::singleton().page()->page();
-    id root = static_cast<PlatformUIElement>(WKAccessibilityRootObject(page));
```

Comment:
We should remove or at least deprecate WKAccessibilityRootObject.  I don't think there are any non-WebKitTestRunner users, so we should be able to remove it.